aedes.js

```diff
@@ -24,6 +24,21 @@ const ports = {
 
 const args = process.argv.filter(arg => ports[arg])
 
+const sockets = new Set()
+
+function addSocket (socket) {
+  sockets.add(socket)
+  socket.on('close', () => {
+    sockets.delete(socket)
+  })
+}
+
+function destroySockets () {
+  for (const s of sockets.values()) {
+    s.destroy()
+  }
```

Comment:
Why is this needed?

Reply:
When I try to kill aedes I will close servers but if there are existing sockets on them they will never close, this is used to keep track of existing sockets and force close them on aedes close